pom.xml

```diff
@@ -195,6 +196,21 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>com.google.api</groupId>
+        <artifactId>api-common</artifactId>
+        <version>${project.api-common.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>com.google.errorprone</groupId>
+            <artifactId>error_prone_annotations</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+          </exclusion>
+        </exclusions>
```

Comment:
What was the reason we need to exclude these deps? Is this failing enforcer errors?

Reply:
Yes, indeed. I added this to the PR description (and release notes).

Reply:
I missed this comment. 
Done, and the exclusions are not necessary anymore, but guava was not bumped up or down, it simply went from `x.x.x-android` to `x.x.x-jre`.
Do you know if this has any implications for android compatibility, or whether we are meant to support android? I see no claim of this in this repository.

Reply:
Is there a reason it must go from `-android` to `-jre`? Can we keep it as `-android` and only bump the version?

I don't know if we promise anything for android compatibility in this repo. But I would prefer to err on the side of caution and keep it the way it is until we do a deeper investigation to swap from `-android` to `-jre`.